core-web/apps/dotcms-ui/src/app/api/services/guards/pages-guard.service.ts

```diff
@@ -0,0 +1,26 @@
+import { map, take } from 'rxjs/operators';
+import { Injectable } from '@angular/core';
+import { CanActivate } from '@angular/router';
+import { Observable } from 'rxjs';
+import { DotPropertiesService } from '../dot-properties/dot-properties.service';
+
+/**
+ * Route Guard that based on DOTFAVORITEPAGE_FEATURE_ENABLE flag allows/denies access to Pages portlet.
+ */
+@Injectable()
+export class PagesGuardService implements CanActivate {
+    constructor(private dotConfigurationService: DotPropertiesService) {}
+
+    /**
+     * Guard checks if DOTFAVORITEPAGE_FEATURE_ENABLE flag is true in dotmarketing-config.properties.
+     * @returns Observable<boolean>
+     */
+    canActivate(): Observable<boolean> {
+        return this.dotConfigurationService.getKey('DOTFAVORITEPAGE_FEATURE_ENABLE').pipe(
```

Comment:
What do you think to add that key to the enum `FeaturedFlags` at `core-web/apps/dotcms-ui/src/app/portlets/shared/models/shared-models.ts`

Reply:
not needed, since this will be always active when it gets released

so for now it's temporary thing

Reply:
Same for Experiments.

Reply:
@oidacra changed!